### PR TITLE
Re-opened pull request

### PR DESCRIPTION
**Pull request by charlesnchr**
Updated readme with instruction on installing with antigen and included the maintenance warning. Otherwise identical to previous pull request, see #6.

### CODE REVIEW

File: autojump.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
+
 import ranger.api
 import subprocess
 from ranger.api.commands import *
+import os
```
Comment (fdw) [inline autojump.py]: Is that import necessary?

File: autojump.py [hunk 2]
```diff
@@ -1,6 +1,8 @@
+
```
Comment (fdw) [inline autojump.py]: Can you please remove that empty line?

**Review by fdw — CHANGES_REQUESTED** [review]
Looks good to me in general, thank you 🙂

If you can change these small things, please, then I'll merge it

File: autojump.py (hunk 1 above)
Comment (charlesnchr) [reply autojump.py]: Good catch — I was using it for os.path temporarily, but ultimately it was not needed.

File: autojump.py (hunk 2 above)
Comment (charlesnchr) [reply autojump.py]: Done.

**Comment by charlesnchr** [comment]
Good comments. They have been addressed.

**Comment by fdw** [comment]
Thank you very much :)

**Pull request merged**